### DISCUSSION TITLE
Class <>z__ReadOnlyArray<T> generated for literal collections is not tagged with CompilerGeneratedAttribute

### DISCUSSION THREAD

**Discussion by ndepend** [dotnet/csharplang#8006] [General]
Class `<>z__ReadOnlyArray<T>` generated for literal collections is not tagged with `System.Runtime.CompilerServices.CompilerGeneratedAttribute`. I think it should.

![image](https://github.com/dotnet/csharplang/assets/511445/32429fb7-e90f-4cda-8e2d-9b2df6d50656)

**Comment by CyrusNajmabadi** [comment]
Closing out as this is not a language concern.

**Reply by jnm2** [reply]
Can we move to Roslyn and mark up for grabs?

**Reply by CyrusNajmabadi** [reply]
I haven't been able to move issues in a while. The git UI pops up nothing for me. :-(

**Comment by ndepend** [comment]
Sorry, I re-opened it here https://github.com/dotnet/roslyn/issues/72539
